client/my-sites/themes/survey/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { translate } from 'i18n-calypso';
+import React from 'react';
```

Comment:
Let's import `useState` instead of the whole shebang.

```suggestion
import { useState } from 'react';
```

Reply:
This feels really really really minor, but I've changed it.